gcp_variant_transforms/options/variant_transform_options.py

```diff
@@ -144,6 +145,16 @@ def add_arguments(self, parser):
     parser.add_argument('--output_table',
                         default='',
                         help='BigQuery table to store the results.')
+    parser.add_argument(
+        '--generate_call_info_table',
```

Comment:
Why are we defaulting this off?  I would rather see this on by default.

Reply:
Yes, ultimately this should be on. We have a follow-up PR that will modify the main table accordingly (e.g., replace the call_name with the sample_id), otherwise the sample_info table alone is not useful. I think we probably should turn on this flag after that.